src/pages/content/voice-and-tone.mdx

```diff
@@ -1,12 +1,110 @@
+
+## Mapping tone
+
+After plotting these user interactions, we outlined four quadrants that were a natural fit for our
+brand tones.
+
+### Low impact and informational
+
+Contextual feedback to help the user. This content appears right when the user needs it: in
+labels, tooltips, and auto-dismissing alerts. It's good to know, but not to remember.
+
+| Tone to use                            | Character            |
+| -------------------------------------- | -------------------- |
+| **Distilled:** Space, precise, focused | An efficient surgeon |
+
+What surgeons know: precise action takes precise communication. Use the Distilled tone to say it quickly
+and clearly. Nuance and charm get in the way of transactional experiences or basic user feedback.
+
+### High impact and informational
+
+These are mindful memos to our user, not directives. Warnings and unsaved changes modals tell our
+users something is up, but leave it up to the user to decide how to act.
+
+| Tone to use                               | Character         |
+| ----------------------------------------- | ----------------- |
+| **Humblident:** Even, Calm, Knowledgeable | A resolute sherpa |
+
+The sherpa knows the way up the mountain, but you’re the one who climbs it. Zendesk’s Humblident
+tone (a proprietary mix of ‘humble’ and ‘confident’) is how we empower users to make choices. We
+don’t ask “Are you sure?” because informed users know what’s right for them.
+
+### High impact and instructional
+
+These are mindful memos to our user, not directives. Warnings and unsaved changes modals tell our
+users something is up, but leave it up to the user to decide how to act.
+
+| Tone to use                          | Character          |
+| ------------------------------------ | ------------------ |
+| **Real:** Direct, Candid, Respectful | Your no B.S bestie |
+
+We love our no B.S. besties because they say what needs to be said—even the hard stuff—from a place
+of empathy. Use the Real tone whenever there’s potential for disappointment. Deprecated feature?
+Hacked account? Skip the sugar-coated corp-speak and never make jokes. Real is the sweet spot
+between Charming and Distilled: tell it like it is.
+
+### Low impact and Instructional
+
+Actionable but adjacent to key user flows. Onboarding and empty states are cool, but not mandatory
+to get work done. Content’s job is to make these experiences worthwhile and delightful for the user.
+
+| Tone to use                                | Character    |
+| ------------------------------------------ | ------------ |
+| **Charming:** Refreshing, Warm, Well-timed | A great host |
+
+A great host knows the right thing to say and how to say it. A sprinkle of Charming tone adds warmth
+(and umami) to the product. It’s perfect for low-stakes interactions where the user has more time to
+read and less to do. But don’t overdo it. The goal is a smile, not a LMFAO. We’re here to serve, not
```

Comment:
Putting the table between these two paragraphs doesn't quite work. It's somewhat hard to read and we are burying the key information: which tone to use. Now that I see it in situ, I realized I should have organized this differently. You can see my proposed revision in this [Google doc](https://docs.google.com/document/d/1TFy548JvyEIQ8-GFF13pJFQwOeUOQ_ri6WNOvKoQdEI/edit?disco=AAAAGreNp88) because it's easier to see there.